docs/docs/operator/OperatorKeys.md

```diff
@@ -0,0 +1,23 @@
+---
+id: "operator-keys"
+title: "Operator Interaction With NodeLicenses"
+sidebar_label: "NodeLicenses Interactions"
+sidebar_position: 6
+custom_edit_url: null
+---
+
+# Operator Interaction With NodeLicenses
+
+The operator interacts with NodeLicenses through the following function:
+
+## loadOperatorKeysFromGraph
+
+• **Description**: Loads all operating keys and metadata from the subgraph. It will find all keys and pool for this operator, including key timestamps, assigned pools and owner kyc status and will also attach submission objects to each sentryKey for processing the last closed challenge.
```

Comment:
Do we know if this includes pools for an operator that has only staked keys or esXai in the pool, but is not the pool owner/delegate?

Reply:
It does not include the pools where the operator has keys staked in, in that case we would only operate that key not all keys staked in the pool.

This description has been rewritten, however considering the bulksubmission updates this is going to be depricated soon.